### PR TITLE
fix: Add the spinner to the button click

### LINKED ISSUE #1113: Remove Device registration and Governance Vue file

### PR DESCRIPTION
**Pull request by talwinder50**
Screen Shot 2021-09-28 at 9 51 26 AM

closes #1113

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>
Screen Shot 2021-09-28 at 10 04 55 AM

### CODE REVIEW

**Comment by talwinder50** [comment]
Addressed the comment : https://github.com/trustbloc/wallet/pull/1131#pullrequestreview-764178354

File: test/mock/demo-login-consent-server/templates/login.html [hunk 1]
```diff
@@ -58,15 +57,27 @@
...
                     id="password" name='password' type="password" value="f00B@r!23">
                         </div>
-
                         <div class="flex items-center justify-center">
-                            <button class="gradient text-sm sm:text-xs rounded-md text-white mb-8 h-10 px-8 shadow-lg" type='submit' id="accept" name='btn_login'>
-                                Sign In
+                            <button class="flex items-center justify-center gradient rounded-md mb-8 h-10 px-8 shadow-lg"
+                                    type='submit'
+                                    id="accept" onclick="signIn()">
+                                <span class="flex-0" id="btn_text"> Sign in </span>
```
Comment (birtony) [inline test/mock/demo-login-consent-server/templates/login.html:64]: Can we remove extra spaces around the text?
```suggestion
                                <span class="flex-0" id="btn_text">Sign in</span>
```

File: test/mock/demo-login-consent-server/templates/login.html [hunk 2]
```diff
@@ -35,12 +35,11 @@
...
     <p class="text-black text-2xl text-center font-bold">Demo Sign In</p>
-    <div class="container mx-auto flex flex-wrap h-auto lg:w-1/3 w-full">
-        <div class="w-full flex flex-col flex-grow flex-shrink">
-            <div class="flex-none mt-auto rounded-b rounded-t-none overflow-hidden">
-                <div class="flex items-center p-14 justify-center">
-                    <form class="bg-white shadow-xl rounded-xl lg:px-11 sm:px-11 px-4 pt-8
-                    md:flex-wrap md:justify-between w-screen h-auto" action="/login" method="post">
+    <div class="container mx-auto h-auto lg:w-1/3 w-full">
+            <div class="mt-auto rounded-b rounded-t-none overflow-hidden">
+                <div class="items-center p-14 justify-center">
```
Comment (birtony) [inline test/mock/demo-login-consent-server/templates/login.html:40]: Is the container not aligned properly horizontally or is it just a part of the screen that was screenshot? I think you're missing `flex` here:
```suggestion
                <div class="flex items-center p-14 justify-center">
```

**Review by birtony — COMMENTED** [review]
It would be helpful to see the loading state on a screenshot too

File: test/mock/demo-login-consent-server/templates/login.html (hunk 2 above)
Comment (talwinder50) [reply test/mock/demo-login-consent-server/templates/login.html:40]: No flex is not required here,  that was just screenshot

Comment (birtony) [reply test/mock/demo-login-consent-server/templates/login.html:40]: If it's not required, then you should remove `items-center` and  `justify-center` - these only apply when the container uses flexbox @talwinder50

Comment (birtony) [reply test/mock/demo-login-consent-server/templates/login.html:40]: But I'm pretty sure we need it and it used to be there too

Comment (talwinder50) [reply test/mock/demo-login-consent-server/templates/login.html]: Yes items-center and justify-center needs to be removed, i have tested it we dont need flex on this div , button dont renders full width if you have flex on all the divs , latest patch works fine

Comment (birtony) [reply test/mock/demo-login-consent-server/templates/login.html]: Alright then. Could you post a screenshot with the loading state, please?

**Review by birtony — APPROVED** [review]
LGTM

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**